samples/display/cfb/src/main.c

```diff
@@ -38,7 +38,7 @@ void main(void)
 
 	if (display_set_pixel_format(dev, PIXEL_FORMAT_MONO10) != 0) {
 		printf("Failed to set required pixel format\n");
-		return;
+		/* return;*/
```

Comment:
the problem is mcux_elcd driver this pixel_format is not support, and for rgb565 panel it seems this is not a must.

Reply:
Actually  due to adding RGB565 to CFB it works, looks to me that this code is just a safety check.
You could first do a call to `display_get_capabilities` to get the supported pixel formats and then set the one you need.

Reply:
ok

Reply:
But with the proposed change it will always print a failure because it is setting a none supported mode and actually the code is setting the wrong mode.

So why not query the modes supported by the display (via `display_get_capabilities`) and apply one that is supported by CFB and if no supported mode is found report the error

Reply:
ok, let me do this way.